_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -1,4 +1,6 @@
 <?php
+use Automattic\Jetpack\Partners\Affiliate as Affiliate;
```

Comment:
I think ` as Affiliate` isn't necessary, is it? If we omit it, we should end up with the same name.

Reply:
Fixed. Thanks.